package.json

```diff
@@ -73,7 +74,7 @@
     "ts-loader": "^9.0.0",
     "typescript": "^3.8.3",
     "walkdir": "^0.4.0",
-    "webpack": "^4.34.0",
+    "webpack": "^5.0.0",
```

Comment:
This sneaked in - I don't think it's related to this PR?

Reply:
Right, I got dependency tree error because `karmar-webpack` ask for `5.0.0`. Without this update, it cause fails on `npm install`.